coderd/database/queries/templateversionparameters.sql

```diff
@@ -0,0 +1,32 @@
+-- name: InsertTemplateVersionParameter :one
```

Comment:
Thoughts on making this insert multiple similar to how we insert logs? It could be a premature optimization though, your call.

Reply:
Good idea, but agree that's a bit premature. Let's leave it for later.